codalab/model/bundle_model.py

```diff
@@ -742,6 +742,11 @@ def update_bundle(self, bundle, update, connection=None):
         # Apply the column and metadata updates in memory and validate the result.
         metadata_update = update.pop('metadata', {})
         bundle.update_in_memory(update)
+
+        # Make individual tags out of space separated words
+        if metadata_update.has_key('tags'):
+            metadata_update['tags'] = metadata_update['tags'].split()
+
```

Comment:
Just a guess, but this will probably throw an `AttributeError` since `metadata_update['tags']` should already be a `list`